language-support/ts/codegen/tests/BUILD.bazel

```diff
@@ -7,12 +7,12 @@ load("@build_environment//:configuration.bzl", "sdk_version")
 load("@os_info//:os_info.bzl", "is_windows")
 
 # This rule builds a dar from the daml sources under the 'daml'
-# directory. It is referenced from the 'build-and-lint' test target that
+# directory. It is referenced from the 'build-and-lint-test' target that
 # follows.
 daml_compile(
-    name = "build-and-lint-1.0.0",
+    name = "build-and-lint",
     srcs = glob(["daml/**/*.daml"]),
-    main_src = "daml/Main.daml",
+    version = "1.0.0",
```

Comment:
Why is this necessary? The daml.yaml specifies the version as 1.0.0 doesn't it?

Reply:
You rely on it. Nowhere else does.
I've got a mind to change the default to something unusual like 1.2.3

Reply:
Something is not right.
```
shaynefletcher:foo$ cat daml.yaml
sdk-version: 0.13.56-snapshot.20200318.3529.0.6ea118d6
name: foo
source: daml
init-script: Setup:initialize
parties:
  - Alice
  - Bob
version: 0.0.1
dependencies:
  - daml-prim
  - daml-stdlib
  - daml-script
```
If I now,
```
daml build
```
I get,
```
Created .daml/dist/foo-0.0.1.dar
```

Reply:
Oh I see what you are getting at. Somehow this test is relying on 1.0.0. That's a mistake. I'll amend it to expect 0.0.1. If I do that will it remove the need for the version string in the invocation of the rule?

Reply:
I got it now. Thank-you!